src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -77,7 +71,7 @@ public FindCommand(ContainsGeneralKeywordsPredicate generalPredicate) {
     public static boolean isValidSpecificFind(List<String> specificKeywords) {
         boolean result = true;
         for (int i = 0; i < specificKeywords.size(); i++) {
-            if (specificKeywords.get(i).matches(VALIDATION_REGEX)) {
+            if (specificKeywords.get(i).isBlank()) {
```

Comment:
I chose to use `.isBlank()` rather than `.isEmpty()` as the former also guards against \t, \n, \r, \f, \s just in case the user somehow manages to get it past the parser.